ui/app/AppLayouts/Wallet/services/dapps/DAppsRequestHandler.qml

```diff
@@ -436,7 +440,7 @@ SQUtils.QObject {
             let feesInfo = getEstimatedMaxFees(data, method, chainId, mainNetChainId)
 
             let totalMaxFees = Math.sum(feesInfo.maxFees, feesInfo.l1GasFee)
-            let maxFeesEth = Math.div(totalMaxFees, Math.fromString("1000000000"))
+            let maxFeesEth = Math.div(totalMaxFees, Math.fromNumber(1, 9))
```

Comment:
Maybe using name `Math` for `SQUtils.AmountArithmetic` was not the best idea, as it usually denotes js static object